src/poprox_storage/repositories/clicks.py

```diff
@@ -35,13 +35,14 @@ def __init__(self, connection):
         super().__init__(connection)
         self.tables = self._load_tables("clicks")
 
-    def track_click_in_database(self, newsletter_id, account_id, article_id):
+    def track_click_in_database(self, newsletter_id, account_id, article_id, created_at):
```

Comment:
In order to avoid breaking the code that calls this function, `created_at` needs a default so that it can be provided but isn't mandatory:
```suggestion
    def track_click_in_database(self, newsletter_id, account_id, article_id, created_at=None):
```

Reply:
Sure, thanks for the suggestion!